flexmeasures/cli/data_add.py

```diff
@@ -1205,14 +1205,20 @@ def add_schedule_for_storage(
         },
     )
     if as_job:
-        job = create_scheduling_job(sensor=power_sensor, **scheduling_kwargs)
+        job = create_scheduling_job(
```

Comment:
The `create_scheduling_job` function asks for `asset_or_sensor` rather than `asset_or_sensor_id`.